crates/database/src/migrations/mod.rs

```diff
@@ -1,19 +1,21 @@
 use crate::pool::DbPool;
 use anyhow::{Context, Result};
-use refinery::embed_migrations;
+use refinery::load_sql_migrations;
 use tracing::info;
 
-// Embed migrations from the migrations folder
-embed_migrations!("src/migrations/sql");
```

Comment:
@PierreLeGuen We switched from embedding migration to loading SQL migrations because the orders of the SQL files in the binary are non-deterministic across machines. Please help review whether there'll be issues with this change.